### ISSUE TITLE
User avatars are not rendered

### ISSUE THREAD

**Issue by Saucistophe** [mattermost/mattermost-plugin-calls#722]
We just tried the plugin after upgrading Mattermost from version 7 to the latest. We also updated the plugin to its latest version.

The plugin works, but somehow the avatars have an incorrect URL ("/api") and are not rendered:

![image](https://github.com/mattermost/mattermost-plugin-calls/assets/7082839/27bbbf6e-feaf-465e-8d88-b82c7c71519a)

Did we miss something during the upgrade?

**Comment by streamer45** [comment]
@Saucistophe Haven't seen that before. Are you seeing the avatars correctly in other components outside of the plugin?

It would be useful to clear the cache and share the output of the Network tab (see example below).

![image](https://github.com/mattermost/mattermost-plugin-calls/assets/1832946/44402885-0ec3-4a4c-b0de-a5be329d4126)

Also, are you running MM under a subpath by any chance?

**Comment by Saucistophe** [comment]
Yes, they are correctly displayed. It comes from the fact that we serve mattermost under a nested route (XXXX.com/mattermost).
The avatars are correctly displayed in the app, with the url:
```
https://<redacted>.com/mattermost/api/v4/users/5f56da3khjnffxiitqscmreaaw/image?_=0
```
But in the plugin they are KO, with url:
```
https://<redacted>.com/api/v4/users/1of93e5337ngjfhg3exeys4hme/image?_=1707915276925
```
notice the missing `/mattermost`

**Comment by streamer45** [comment]
@Saucistophe Could you confirm the Calls plugin version you are running?

**Comment by streamer45** [comment]
Thanks for the report @Saucistophe . A fix for this should be available in the upcoming release (v0.27)

**Comment by Saucistophe** [comment]
Thanks for the quick response!!
